src/components/CreateNewListModal/component.js

```diff
@@ -4,21 +4,30 @@ import { Field } from 'formik';
 import {
   Modal,
   Form,
-  Typography,
-  Icon
+  Typography
 } from 'antd';
 
-import InputField from '../../../components/InputField';
+import InputField from '../InputField';
 
 const CreateNewListModal = ({
   showModal,
   modalVisible,
   hideModal,
   handleSubmit,
-  errors
+  errors,
+  triggerComponent: TriggerComponent,
+  triggerProps
 }) => (
   <>
-    <Icon onClick={showModal} type="plus-circle" />
+    <TriggerComponent
+      {...triggerProps} /* eslint-disable-line react/jsx-props-no-spreading */
+      onClick={() => {
```

Comment:
Don't use inline functions in components (UI part), move this to container

Reply:
Done